fdbserver/workloads/WriteTagThrottling.actor.cpp

```diff
@@ -0,0 +1,229 @@
+/*
+ * WriteThrottling.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//
+// Created by sfc-gh-xwang on 6/29/20.
```

Comment:
Are we going to add this as header now? I'm totally fine with it. Just want to be consistent. :)

Reply:
I don't think so - it's also not necessary as git will keep the history :) (I would be fine having a header like this if people want that, but I agree that it should be consistent and there should be consensus first).

@sfc-gh-xwang can you please remove these?

Reply:
No, this just the auto-generated head. However, I guess this would be helpful if this test case becomes obsoleted or cannot run, some can ping me to fix.

Reply:
Sure thing.